components/brave_rewards/resources/extension/brave_rewards/background/reducers/rewards_panel_reducer.ts

```diff
@@ -61,6 +62,10 @@ export const rewardsPanelReducer = (state: RewardsExtension.State | undefined, a
       state.walletCreating = false
       state.walletCreated = false
```

Comment:
You need to set `state.walletCorrupted` to false here

Reply:
same in `CREATE_WALLET`

Reply:
Done.